blazor/HostedExample/HostedExample/Client/Views/FetchDataView.razor.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Reactive.Linq;
+using System.Threading.Tasks;
+using HostedExample.Client.ViewModels;
+using Microsoft.AspNetCore.Components;
+using ReactiveUI;
+
+namespace HostedExample.Client.Views
+{
+    public partial class FetchDataView : IViewFor<FetchDataViewModel>
+    {
+        [Inject]
+        public FetchDataViewModel ViewModel { get; set; }
+
+
+        object IViewFor.ViewModel
+        {
+            get => ViewModel;
+            set => ViewModel = (FetchDataViewModel)value;
+        }
+
+        protected override async Task OnInitializedAsync()
+        {
+            await ViewModel.LoadForecasts.Execute().SubscribeOn(RxApp.MainThreadScheduler);
```

Comment:
Ditto

Reply:
I can avoid SubscribeOn in Counter but FetchData _**must**_ be awaited or the page doesn't load.  I'd like to leave this one in.

Reply:
Done via ToTask()